provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -335,6 +335,7 @@ data:
         "aci-vmm-domain": {{config.aci_config.vmm_domain.domain|json}},
         "aci-vmm-controller": {{config.aci_config.vmm_domain.controller|json}},
         "aci-policy-tenant": {{config.kube_config.default_endpoint_group.tenant|json}},
+        "app-profile": {{ config.kube_config.default_endpoint_group.app_profile|json }},
```

Comment:
Should this be aci-app-profile to follow the naming convention of the other properties?

Reply:
sure, will change that